src/relay/backend/interpreter.cc

```diff
@@ -18,7 +18,7 @@
  */
 
 /*!
- *  Copyright (c) 2018 by Contributors
+ *  Copyright (c) 2019 by Contributors
```

Comment:
In my experience such changes are usually a year range. does the ASF not need a year range here ?
git blame suggests this file was created in 2018, thus this should read copyright 2018-19 IIUC. However IANAL and the exact definition depends on project policy.

@tqchen , any thoughts

Reply:
Thank you for catching this! I just wrongly assume to use the latest year whenever I update a file.